modules/retail/pages/retail-install-unified.adoc

```diff
@@ -14,18 +14,52 @@ For information on registering with {scc}, or obtaining installation media, see
 . Boot your server from the installation image, you might need to adjust the boot order in the BIOS.
 . When prompted, select [guimenu]``Installation``.
 +
-. In the [guimenu]``Language, Keyboard and Product Selection`` screen, check the [guimenu]``SUSE Manager Retail Server`` checkbox, and click btn:[Next].
+. In the [guimenu]``Language, Keyboard and Product Selection`` screen, check the [guimenu]``SUSE Manager Server`` checkbox, and click btn:[Next].
 +
 image::unifiedinstaller-install.png[scaledwidth=80%]
 . Read and agree to the End User Licence Agreement, and click btn:[Next].
 . In the [guimenu]``Registration`` screen, check the [guimenu]``Register System via scc.suse.com`` checkbox, enter your {scc} credentials, and click btn:[Next].
 . OPTIONAL: In the [guimenu]``Add On Product`` screen, select any additional or add-on products you require, and click btn:[Next].
-. In the [guimenu]``System Role`` screen, check the [guimenu]``SUSE Manager Retail Server`` checkbox, and click btn:[Next].
+. In the [guimenu]``System Role`` screen, check the [guimenu]``SUSE Manager for Retail Server`` checkbox, and click btn:[Next].
 . In the [guimenu]``Suggested Partitioning`` screen, accept the default values, or use the btn:[Guided Setup] or btn:[Expert Partitioner] options to customize your partitioning model, and click btn:[Next].
 . In the [guimenu]``Clock and Time Zone`` screen, enter your region and timezone, and click btn:[Next].
 . In the [guimenu]``Local Users`` screen, create a new user, and click btn:[Next].
 . In the [guimenu]``System Administrator "root"`` screen, create the "root" user, and click btn:[Next].
-. Review the settings on the [guimenu]``Installation Settings`` screen, and then click btn:[Install].
+. In the [guimenu]``Installation Settings`` screen ensure that SSH access is open.
+Review the settings and click btn:[Install].
+
+
+
+.Procedure: Run the Installation Script on the {susemgr} Server
+. Use SSH to access the command prompt of the {susemgr} Server.
+. Run the installation script:
++
+----
+yast susemanager_setup
+----
+. Follow the prompts to set up your account.
+Ensure you take note of the passwords you set, you will need them later on. 
+
+
+
+.Procedure: Installing {susemgr} Branch Server from a DVD image
+
+. Boot your server from the installation image, you might need to adjust the boot order in the BIOS.
+. When prompted, select [guimenu]``Installation``.
++
+. In the [guimenu]``Language, Keyboard and Product Selection`` screen, check the [guimenu]``SUSE Manager Retail Branch Server`` checkbox, and click btn:[Next].
```

Comment:
Above you changed that step from "SUSE Manager Retail Server" to "SUSE Manager Server". Thus I'm wondering whether you can actually select "SUSE Manager Retail Branch Server" here?

Reply:
That was following along with Lukas' video. I hadn't realised how wrong we got that ;)

Screenshot from the vid:
![image](https://user-images.githubusercontent.com/3914967/60640725-ddb7a480-9e6b-11e9-953b-e92bc09fe9ef.png)